src/test/java/tests/comments/GetCommentTestAPI.java

```diff
@@ -0,0 +1,38 @@
+package tests.comments;
+import apimethods.CommentAPI;
+import common.TestBase;
+import constants.KeyParameters;
+import io.restassured.response.Response;
+import models.comments.CommentRequest;
+import models.posts.PostRequest;
+import models.products.ProductRequest;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import apimethods.PostAPI;
+import apimethods.ProductAPI;
+import static constants.KeyParameters.ID;
+import static constants.KeyParameters.POST_ID;
+
+public class GetCommentTestAPI extends TestBase {
+    private Integer commentId;
+    private Integer postId;
+    private Integer relatedProductId;
+
+    @BeforeTest
+    public void prepareData() {
+        relatedProductId = ProductAPI.createProduct(new ProductRequest("Falcon", 10, "EUR")).path(KeyParameters.ID);
```

Comment:
da spojimo u metodu koja vraća commentId i postId. Da zajedno razmislimo kako to možemo a da ne koristimo hashMape :)